### PR TITLE
New language: Arduino

### PR DESCRIPTION
**Pull request by Sh4rk-Byte**
Add arduino language support, since arduino is basically cpp, the chip refers to cpp.

### CODE REVIEW

File: languages.yaml [hunk 1]
```diff
@@ -79,6 +79,25 @@ Ada:
...
+    {0} -===:.     .-====..-===-:      :==== 
+    {0}  :============-.    .-============:  
+    {0}     .--=====-:         .--=====-:               
+  colors:
+    ansi:
+      - cyan
+    hex:
+      - '#189BA1'
+    chip:
+      - "#F34B7D"
```
Comment (spenserblack) [inline languages.yaml:100]:
```suggestion
    chip: "#F34B7D"
```
This value isn't an array.

**Review by spenserblack — CHANGES_REQUESTED** [review]
Almost there!

File: languages.yaml [hunk 2]
```diff
@@ -79,6 +79,22 @@ Ada:
...
+    {0}===  .----   :===-   :==-.  ===
+    {0}-==.        :=====-   .:   .==-
+    {0} -==-.. .:-===: :===-:. ..-=== 
+    {0}  .-=======-.     .-=======-:      
+  colors:
+    ansi:
+      - cyan
+    hex:
+      - '#189BA1'
+    chip: "#F34B7D"
```
Comment (o2sh) [inline languages.yaml:97]: Hi @Sh4rk-Byte 

I cleaned your design a bit, tell me what you think:

image

the code:

```
Arduino:
  type: programming
  ascii: |
    {0}     ,=======.    ,=======.
    {0}    //       \\  //       \\
    {0}   //         \\//    #    \\
    {0}   ||   ###    //    ###   ||
    {0}   \\         //\\    #    //
    {0}    \\       //  \\       //
    {0}     `======='    `======='
    {0}      __   __       ___       __
    {0} /\  |__) |  \ |  |  |  |\ | /  \
    {0}/--\ |  \ |__/ \__/ _|_ | \| \__/
  colors:
    ansi:
      - cyan
    hex:
      - "#00838B"
    chip: "#F34B7D"
```

Comment (spenserblack) [reply languages.yaml:99]: @o2sh FYI I just expanded the ASCII art in https://github.com/o2sh/onefetch/pull/1176/commits/0aa16511eb1d30e59c6ce57887dd1d56222204d0. I think @Sh4rk-Byte shrunk it to handle the "line to large" compilation error, but it was triggered by trailing whitespace, not a problem with the actual art.

**Review by spenserblack — APPROVED** [review]
Don't have any more changes to request myself, but I'll leave it up to @o2sh to review styling.

File: languages.yaml (hunk 2 above)
Comment (Sh4rk-Byte) [reply languages.yaml:99]: @o2sh yeah I like your ASCII art, it's much clearer. Thanks

**Review by o2sh — APPROVED** [review]
Thanks for your first contribution @Sh4rk-Byte and @spenserblack for the help!

🎉 ❤️

**Comment by spenserblack** [comment]
If you would like to help with #1152, we would appreciate your help [over at gengo](https://github.com/spenserblack/gengo). What's most important to contribute is your familiarity with the language, so, even if you don't make a PR, [you can make an issue](https://github.com/spenserblack/gengo/issues/new?assignees=&labels=new-language&projects=&template=new-language.yml&title=Language%3A+). And if you do, I'll add you to [the credits](https://github.com/spenserblack/gengo/blob/main/CREDITS.md) :wink:

**Pull request merged**